build/upgrade_pdfjs.sh

```diff
@@ -24,52 +23,40 @@ mkdir ${DOC_STATIC_ASSETS_PATH}
 \cp -R src/third-party/doc/${DOC_CURRENT_ASSETS_VERSIONS}/exif.js ${DOC_STATIC_ASSETS_PATH}/
 \cp -R src/third-party/doc/${DOC_CURRENT_ASSETS_VERSIONS}/exif.min.js ${DOC_STATIC_ASSETS_PATH}/
 \cp -R src/third-party/doc/${DOC_CURRENT_ASSETS_VERSIONS}/images ${DOC_STATIC_ASSETS_PATH}/images
-
-echo "-----------------------------------------------------------------------------------"
-echo "Cloning pdfjs-dist repo at branch: $DOC_STATIC_ASSETS_BRANCH..."
-echo "-----------------------------------------------------------------------------------"
-rm -rf ./pdfjs-dist/
-git clone https://github.com/mozilla/pdfjs-dist.git --depth 1 --single-branch --branch ${DOC_STATIC_ASSETS_BRANCH}
-
+\cp -R src/third-party/doc/${DOC_CURRENT_ASSETS_VERSIONS}/cmaps ${DOC_STATIC_ASSETS_PATH}/cmaps
```

Comment:
Do we need a solution to retrieve updated `cmaps`, as well? We're only copying over the previous version at the moment.

Reply:
Did some more research and it looks like https://cdn.jsdelivr.net/npm/pdfjs-dist/ offers everything we need including `cmaps` and a minified version of `pdf_viewer`. I'm going to switch to that

Reply:
I'm concerned about introducing a net new third-party into our artifact chain of custody. Do we use that service anywhere else?

Reply:
I've opted to go with https://cdn.jsdelivr.net/npm/pdfjs-dist/ here. cdn.cloudflare didn't offer the legacy build package, and the prebuild from github didn't include `pdf_viewer.mjs`.

I think its relatively safe given that its a recommended cdn to use by pdf.js and we only use it during the upgrade script. So we are not dependent on it to run the preview sdk